app/image/suite_test.go

```diff
@@ -0,0 +1,35 @@
+package image
```

Comment:
There is already a test suite declared on this package. It's inside the `app/image/image_test.go` file. You don't need this file.